app/test/integration-test/README.md

```diff
@@ -83,36 +85,45 @@ The following system properties (or environment variables) are known to the proj
 
 ## Running tests
 
-You can run the tests from your favorite Java IDE (e.g. Eclipse, IntelliJ) as normal JUnit test. Also you can run all available tests with
-Maven build tool:
+Test sources are located in `/app/test/integration-test`. As the tests are nothing but normal JUnit tests you can run
+those from your favorite Java IDE (e.g. Eclipse, IntelliJ) or via mMaven build.
+
+The easiest way to get the tests executed is to use the Syndesis build tool:
 
 ```bash
-mvn verify
+tools/bin/syndesis integration-test
 ```
 
 This will execute all available integration tests. You can also run single tests or test methods. Just give the test class name and/or test method name as
 an argument.
 
 ```bash
-mvn verify -Dit.test=MyTestClassName
+tools/bin/syndesis integration-test -t MyTestClassName
 ```
 
 ```bash
-mvn verify -Dit.test=MyTestClassName#mytestMethodName
+tools/bin/syndesis integration-test -t MyTestClassName#mytestMethodName
 ```
 
-Please, notice that the integration test may have been disabled by default through the setting of the variable `skip.integration.tests` to `true`. If that is the case, ensure to override the value to `false` instead:
```

Comment:
I undestand that property will be implicit in the `syndesis integration-test` command execution, right?

Reply:
Yep